frontend/src/components/widget/Checkbox.js

```diff
@@ -156,7 +156,8 @@ const Checkbox = (props) => {
  * @prop {string} [widgetField]
  * @prop {string|number} [id]
  * @prop {func} [updateItems] - function used for updating the filter items before having an active filter
- * @prop {bool} [isEdited] - this flag is set only for checkboxes in Tabs
+ * @prop {bool} [isEdited] - this flag is set for checkboxes in Tabs/Attributes, where we have additional
+ *                           requests which would otherwise break the current flow
```

Comment:
ok... but what precisely does that flag do?

e.g. IsEdited is true when the component was edited by user but... bla bla bla.

If we cannot phrase it like that it's an indicator of bad design/workaround.